ccm_web/package.json

```diff
@@ -49,6 +49,7 @@
     "morgan": "^1.10.0",
     "mysql2": "^2.3.2",
     "notistack": "^1.0.10",
+    "p-limit": "^3.1.0",
```

Comment:
There is a more recent version of `p-limit`, version `4`, but it uses ESM, which would require some tinkering with `package.json` and `tsconfig`s, and maybe more dramatic import changes across the project. This should probably be handled separately, and it may not be urgent. I need to do more reading about it. See https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c and https://www.typescriptlang.org/docs/handbook/esm-node.html

Reply:
> but it uses ESM
What are we currently supporting now if it is not ECMA Script module? I felt we are Pardon me for my ignorance.

Reply:
I'm not an expert on this, but I believe CommonJS is the answer. There is some interoperability between module systems, but I got an error when trying to run `4.0.0` in the backend. The resources I linked to should provide more info. I will try this separately, I believe @zqian also commented on this at one point. While NodeJS seems to have implemented ESM, TypeScript seems to be flagging their support as experimental.

Reply:
See #330.